projects/1.19/assets/ad-astra/ad_astra/lang/zh_cn.json

```
@@ -0,0 +1,728 @@
+{
+  "advancements.ad_astra.ad_astra.description": "§b建造一个NASA工作台以制造火箭",
+  "advancements.ad_astra.ad_astra.title": "Ad Astra",
+  "advancements.ad_astra.astro_cuisine.description": "§b将生的食物扔在大气层，并利用摩擦生热将其烹制熟。",
+  "advancements.ad_astra.astro_cuisine.title": "§b太空厨房",
+  "advancements.ad_astra.challenger.description": "§b收集每一个星球仪",
+  "advancements.ad_astra.challenger.title": "挑战者",
+  "advancements.ad_astra.glacio_dungeon.description": "§b进入一个霜原地牢",
+  "advancements.ad_astra.glacio_dungeon.title": "勇往直下",
+  "advancements.ad_astra.interstellar.description": "§b进入一个全新的星系",
+  "advancements.ad_astra.interstellar.title": "星际穿越",
+  "advancements.ad_astra.jet_suit.description": "§b穿上喷气式航天服，体验超乎想象的飞行体验。",
+  "advancements.ad_astra.jet_suit.title": "这才叫升级！",
+  "advancements.ad_astra.mars.description": "§b登陆火星",
+  "advancements.ad_astra.mars.title": "红色星球",
+  "advancements.ad_astra.mars_dungeon.description": "§b进入一个火星地牢",
+  "advancements.ad_astra.mars_dungeon.title": "灰尘之下掩埋了什么？",
+  "advancements.ad_astra.mercury.description": "§b登陆水星",
+  "advancements.ad_astra.mercury.title": "晒的真黑",
+  "advancements.ad_astra.mercury_dungeon.description": "§b进入一个水星地牢",
+  "advancements.ad_astra.mercury_dungeon.title": "回到未来",
+  "advancements.ad_astra.moon.description": "§b登陆月球",
+  "advancements.ad_astra.moon.title": "人类的一小步",
+  "advancements.ad_astra.moon_dungeon.description": "§b进入一个月球地牢",
+  "advancements.ad_astra.moon_dungeon.title": "埋藏的秘密",
+  "advancements.ad_astra.netherite_space_suit.description": "§b穿上下界合金航天服，在极端环境下得以生存。",
+  "advancements.ad_astra.netherite_space_suit.title": "抵御严暑",
+  "advancements.ad_astra.rocket_destroyed.description": "§b发射一艘没有宇航员的火箭，并让它在空中爆炸解体。",
+  "advancements.ad_astra.rocket_destroyed.title": "§b追上旅行者1号了吗……或许还没有",
+  "advancements.ad_astra.space_suit.description": "§b穿上你的第一套航天服",
+  "advancements.ad_astra.space_suit.title": "宇航员",
+  "advancements.ad_astra.tier_1_rocket.description": "§b建造一阶火箭",
+  "advancements.ad_astra.tier_1_rocket.title": "艰难的开始",
+  "advancements.ad_astra.tier_2_rocket.description": "§b建造二阶火箭",
+  "advancements.ad_astra.tier_2_rocket.title": "准备，启航火星",
+  "advancements.ad_astra.tier_3_rocket.description": "§b建造三阶火箭",
+  "advancements.ad_astra.tier_3_rocket.title": "未雨绸缪",
+  "advancements.ad_astra.tier_4_rocket.description": "§b建造四阶火箭",
+  "advancements.ad_astra.tier_4_rocket.title": "天空即为极限",
+  "advancements.ad_astra.venus.description": "§b登陆金星",
+  "advancements.ad_astra.venus.title": "远离家乡的炎狱",
+  "advancements.ad_astra.venus_dungeon.description": "§b进入一个金星地牢",
+  "advancements.ad_astra.venus_dungeon.title": "炎狱的宝藏",
+  "biome.ad_astra.glacio_ice_peaks": "霜原冰刺之地",
+  "biome.ad_astra.glacio_snowy_barrens": "霜原覆雪荒地",
```

```suggestion
  "biome.ad_astra.glacio_snowy_barrens": "霜原积雪荒地",
```
1182~1193原版的snowy一般是`积雪的`，1194之后改为`积雪`
不过`的`加不加应该都随便吧